### PR TITLE
feat(20778): add audit log events

### PR DESCRIPTION
**Pull request by nnorbert**
Add audit log events:
- create/update provider
- create/update/delete payment request
- create/update transaction

### CODE REVIEW

File: apps/payments-api/plugins/entities/paymentRequest/index.ts [hunk 1]
```diff
@@ -205,6 +205,24 @@ const buildDeletePaymentRequest =
     }
   };
 
+const buildGetOrganizationIdFromPaymentRequest =
+  (repo: PaymentRequestRepo, log: FastifyBaseLogger, httpErrors: HttpErrors) =>
+  async (paymentRequestId: string): Promise<{ organizationId: string }> => {
+    let result;
+    console.log(">>> ", paymentRequestId);
```
Comment (peschina) [inline apps/payments-api/plugins/entities/paymentRequest/index.ts:212]: do we need this console log?

Comment (alfonsograziano) [inline apps/payments-api/plugins/entities/paymentRequest/index.ts:212]: Cl

**Review by peschina — APPROVED** [review]
LGTM, one comment

File: apps/payments-api/plugins/entities/paymentRequest/index.ts (hunk 1 above)
Comment (nnorbert) [reply apps/payments-api/plugins/entities/paymentRequest/index.ts]: Done

Comment (nnorbert) [reply apps/payments-api/plugins/entities/paymentRequest/index.ts]: Done

**Review by alfonsograziano — APPROVED** [review]

**Pull request merged**